primitives/storage/src/lib.rs

```diff
@@ -334,6 +388,11 @@ impl ChildTrieParentKeyId {
 	}
 }
 
+/// Map of child trie information stored by `ChildInfo`.
+pub type ChildrenMap<T> = BTreeMap<ChildInfo, T>;
+
+const DEFAULT_CHILD_TYPE_PARENT_PREFIX: &'static [u8] = b":child_storage:default:";
```

Comment:
Please move to `well_known_keys`.

Reply:
It was my intention to not exposte it in `well_known_keys`, but you are not the first one that want it back (actually Basti restores it and I put it away again here). My rational is that we should check against the child info api (I did the change at some other place), having some magic constant is not what I like the best, but let's move it back (I am keeping the changed check though).